libs/SmartSync/src/com/salesforce/androidsdk/smartsync/app/SmartSyncSDKManager.java

```diff
@@ -115,6 +118,7 @@ public static void initNative(Context context, KeyInterface keyImpl,
      */
     public static SmartSyncSDKManager getInstance() {
```

Comment:
Any app compiled with smartstore/smartsync will use SmartSyncSDKManager.
Instead you should set that flag from SyncManager.java getInstance